eZ/Publish/Core/Repository/Repository.php

```diff
@@ -802,7 +802,6 @@ public function getSearchService()
         $this->searchService = new SearchService(
             $this,
             $this->searchHandler->contentSearchHandler(),
```

Comment:
Assuming we'll handle only Content based domains, then this does make sense.
I'll update the PRs.